facade/nodeFacade.go

```diff
@@ -169,11 +161,6 @@ func (nf *nodeFacade) TpsBenchmark() statistics.TPSBenchmark {
 	return nf.tpsBenchmark
 }
 
-// StartBackgroundServices starts all background services needed for the correct functionality of the node
-func (nf *nodeFacade) StartBackgroundServices() {
```

Comment:
:+1:  maybe remove this function from the disabled implementation?

Reply:
done